packages/@okta/vuepress-site/docs/reference/api/oie-policies/index.md

```diff
@@ -0,0 +1,864 @@
+    "_links": {
+        "self": {
+            "href": "https://${yourOktaDomain}/api/v1/policies/rst10xzYOyK9ux6v70g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        },
+        "rules": {
+            "href": "https://${yourOktaDomain}/api/v1/policies/rst10xzYOyK9ux6v70g4/rules",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        }
+    }
+}
+```
+### Sample Operation: Get Rule
+
+<ApiOperation method="get" url="/policies/${policyId}/rules/${ruleId}" />
+
+Given a `policyId` identifier and a `ruleId`, returns a [Rule Object](#okta-identity-engine-policy-and-rule-objects).
+
+#### Request Body
+
+| Property | Type   | Description                          |
+|----------|--------|--------------------------------------|
+| policyId | String | The unique identifier of the Policy. |
+| ruleId   | String | The unique identifier of a Rule.     |
+
+#### Request Query Parameters
+
+None
+
+#### Response Body
+
+A [Rule Object](#okta-identity-engine-policy-and-rule-objects).
+
+#### Usage Example
+
+##### Request
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/${policyId}/rules/${ruleId}"
+```
+
+##### Response
+
+```json
+{
+    "name": "Catch-all Rule",
+    "id": "rul10y0LatXn0HP2p0g4",
+    "type": "Okta:IdpRouting",
+    "priority": 0,
+    "conditions": [],
+    "action": "ALLOW",
+    "requirement": {
+        "idpId": "OKTA",
+        "type": "okta_idp"
+    },
+    "status": "ACTIVE",
+    "default": false,
+    "_links": {
+        "self": {
+            "href": "https://${yourOktaDomain}/api/v1/policies/rst10xzYOyK9ux6v70g4/rules/rul10y0LatXn0HP2p0g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT",
+                    "DELETE"
+                ]
+            }
+        },
+        "deactivate": {
+            "href": "https://${yourOktaDomain}/api/v1/policies/rst10xzYOyK9ux6v70g4/rules/rul10y0LatXn0HP2p0g4/lifecycle/deactivate",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        },
+        "policy": {
+            "href": "https://${yourOktaDomain}/api/v1/policies/rst10xzYOyK9ux6v70g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        }
+    }
+}
+```
+
+### Syntax for All Other Operations
+
+See <https://developer.okta.com/docs/reference/api/policy/#policy-api-operations> for request syntax for all other operations.
+
+## Okta Identity Engine Policy and Rule Objects
+
+Okta Identity Engine introduces a set of new Policy and Rule objects that are different from the existing Policy and API objects:
```

Comment:
`Okta Identity Engine introduces a set of new Policy and Rule objects that are different from the existing Policy and API objects` -> `Okta Identity Engine introduces a set of new Policy and Rule objects that are different from the existing Policy and Rule objects`